src/main/java/com/iota/iri/model/AbstractHash.java

```diff
@@ -11,12 +11,24 @@
 import com.iota.iri.storage.Indexable;
 import com.iota.iri.utils.Converter;
 
+/**
+ * Base implementation of a hash object
+ */
 public abstract class AbstractHash implements Hash, Serializable {
     private final Object lock = new Object();
 
+    /**Instantiates null safe byte and trit arrays*/
```

Comment:
We are not instantiating anything here...
If you want to write docs for these `private` members (you don't have to) please explain that they are for thread safety due to [double-check locking](https://en.wikipedia.org/wiki/Double-checked_locking).

Reply:
I think it will suffice to write class docs though and have no private members docs (your call)